tests/components/unifi/test_sensor.py

```diff
@@ -737,13 +737,36 @@ async def test_outlet_power_readings(
     assert outlet_2.attributes.get(ATTR_DEVICE_CLASS) == SensorDeviceClass.POWER
     assert outlet_2.state == "73.827"
 
+    power_budget_reg_entry = ent_reg.async_get(
```

Comment:
One sensor one test, will help simplify later on should we refactor it to use pytest parametrize. If these are very similar, feel free to use parametrize on this test.

Reply:
I'm pretty sure I hate the approach I took with parameterized tests, do you have any pointers?